pkg/manifest/manifest.go

```diff
@@ -26,10 +26,14 @@ const (
 	featureSetAnnotation  = "release.openshift.io/feature-set"
 )
 
-var knownFeatureSets = sets.String{}
+var knownFeatureSets = sets.Set[string]{}
 
 func init() {
-	for featureSet := range configv1.FeatureSets {
+	// map[FeatureSet]*FeatureGateEnabledDisabled{
+	// map[ClusterProfileName]map[FeatureSet]*FeatureGateEnabledDisabled
+	allFeatureSets := configv1.AllFeatureSets()
+	// TODO: should we read cluster profile?
+	for featureSet := range allFeatureSets[configv1.SelfManaged] {
```

Comment:
should iterate over both levels of the map and include all featureSets for this.

Reply:
done